src/components/PageHeaderCompact/PageHeaderCompact.tsx

```diff
@@ -84,7 +84,6 @@ export const PageHeaderCompact = ({
         <div className="cc-page-header-compact__authors">
           {authors.map(author => (
             <Author
-              imageSizes="(min-width: 48em) 6.25rem, 3.125rem"
```

Comment:
Out of interest, why are you removing this?

Reply:
This is for the author(s) at the top of an article. An image here has the same size across all breakpoints (80px)

![Screenshot 2021-09-01 at 10 19 14](https://user-images.githubusercontent.com/10700103/131646450-44f32b99-d64f-432a-93b9-e50fc959c087.png)